WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -969,6 +1010,26 @@ private extension EditableOrderViewModel {
             DDLogError("⛔️ Error fetching product variations for order: \(error)")
         }
     }
+
+    /// Summary of coupon lines
+    /// - Parameter couponLines: order's coupon lines
+    /// - Returns: Coupon codes comma separated to display as a summary
+    func summarizeCoupons(from couponLines: [OrderCouponLine]) -> String? {
+        guard couponLines.isNotEmpty else {
+            return nil
+        }
+
+        let output = couponLines.reduce("") { (output, line) in
+            let prefix = output.isEmpty ? "" : ","
+            return output + prefix + line.code
+        }
```

Comment:
👀 This results in coupon codes being separated by commas without space. We should add space behind the comma `", "` instead. This can also be simplified like so:

```suggestion
        let output = String(couponLines.map { $0.code }.joined(by: ", "))
```

Reply:
Good idea. Done in 7839f79977d0f473caace4b8141ca9769f2b8eb3